lib/renderer/inspector.js

```diff
@@ -4,10 +4,19 @@ window.onload = function () {
   // Use menu API to show context menu.
   window.InspectorFrontendHost.showContextMenuAtPoint = createMenu
 
+  // correct for Chromium returning undefined for filesystem
+  window.Persistence.FileSystemWorkspaceBinding.completeURL = completeURL
+
   // Use dialog API to override file chooser dialog.
   window.UI.createFileSelectorElement = createFileSelectorElement
 }
 
+// Extra / is needed as a result of MacOS requiring absolute paths
+function completeURL (project, path) {
+  project = 'file://'
+  return `${project}/${path}`
```

Comment:
This will _almost certainly_ not work on macOS, can we inject this method on on macOS?